### DISCUSSION TITLE
What are the general requirements for Reanimated 3.0.0

### DISCUSSION THREAD

**Discussion by FN-FAL113** [software-mansion/react-native-reanimated#4132] [Q&A]
The latest release for reanimated 3 is out

**Comment by FN-FAL113** [comment]
just saw the docs and it only shows react native version requirement, I guess that suffice it.

**Comment by ponikar** [comment]
Is anyone able to use React native reanimated 3 with expo?

**Reply by Aris-ngoy** [reply]
it crashes on expo 48
